HPXMLtoOpenStudio/resources/defaults.rb

```diff
@@ -5736,6 +6135,625 @@ def self.get_ceiling_fan_months(weather)
+      branch_circuit.max_current_rating_isdefaulted = true
+    end
+
+    return branch_circuit
+  end
+
+  # Gets the default power rating for a service feeder based on load type, voltage, amps, and attached components.
+  #
+  # @param runner [OpenStudio::Measure::OSRunner] Object typically used to display warnings
+  # @param hpxml_bldg [HPXML::Building] HPXML Building object representing an individual dwelling unit
+  # @param service_feeder [HPXML::ServiceFeeder] Object that defines a single electric panel service feeder
+  # @param default_panels_csv_data [Hash] { load_name => { voltage => power_rating, ... }, ... }
+  # @param electric_panel [HPXML::ElectricPanel] Object that defines a single electric panel
+  # @return [Double] power rating (W)
+  def self.get_service_feeder_power_default_values(runner, hpxml_bldg, service_feeder, default_panels_csv_data, electric_panel, unit_num)
+    type = service_feeder.type
+    component_ids = service_feeder.component_idrefs
+    watts = 0
+
+    if type == HPXML::ElectricPanelLoadTypeHeating
+      hpxml_bldg.heating_systems.each do |heating_system|
+        next if !component_ids.include?(heating_system.id)
+        next if heating_system.is_shared_system
+        next if heating_system.fraction_heat_load_served == 0
+
+        branch_circuit = get_or_add_branch_circuit(electric_panel, heating_system, unit_num)
+
+        if heating_system.heating_system_fuel == HPXML::FuelTypeElectricity
+          watts += UnitConversions.convert(HVAC.get_heating_input_capacity(heating_system.heating_capacity, heating_system.heating_efficiency_afue, heating_system.heating_efficiency_percent), 'btu/hr', 'w')
+        end
+
+        watts += HVAC.get_blower_fan_power_watts(heating_system.fan_watts_per_cfm, heating_system.heating_airflow_cfm)
+        watts += HVAC.get_pump_power_watts(heating_system.electric_auxiliary_energy)
+
+        if branch_circuit.occupied_spaces.nil?
+          branch_circuit.occupied_spaces = get_breaker_spaces_from_power_watts_voltage_amps(watts, branch_circuit.voltage, branch_circuit.max_current_rating)
+          branch_circuit.occupied_spaces_isdefaulted = true
+        end
+      end
+
+      hpxml_bldg.heat_pumps.each do |heat_pump|
+        next if !component_ids.include?(heat_pump.id)
+        next if heat_pump.fraction_heat_load_served == 0
+
+        branch_circuit_odu = get_or_add_branch_circuit(electric_panel, heat_pump, unit_num)
+        branch_circuit_ahu = get_or_add_branch_circuit(electric_panel, heat_pump, unit_num, true)
+
+        watts_ahu = HVAC.get_blower_fan_power_watts(heat_pump.fan_watts_per_cfm, heat_pump.heating_airflow_cfm)
+        watts_odu = HVAC.get_dx_coil_power_watts_from_capacity(UnitConversions.convert(heat_pump.heating_capacity, 'btu/hr', 'kbtu/hr'), branch_circuit_odu.voltage)
+
+        if heat_pump.backup_type == HPXML::HeatPumpBackupTypeIntegrated
+
+          if heat_pump.simultaneous_backup # sum; backup > compressor
```

Comment:
See my previous comment about a misunderstanding. I doubt summing is the right behavior.

Reply:
I'll remove this, too.